cornice/ext/wtforms.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+def validate_wtforms_schema(form_schema, with_matchdict=False):
```

Comment:
Since the module is already named "wtforms" I believe it's enough to name this `validate_schema`.